docs(nginx): Version update and logging clarification (#31639)
DataDog/documentation: content/en/tracing/trace_collection/proxy_setup/nginx.md

```sh
sudo nginx -s reload
```

### Step 2: Configure the log pipeline to parse the trace and span IDs

> The trace and span IDs in the logs will be in hexadecimal format:

Reviewer: ```suggestion
By default, logs capture `dd.trace_id` and `dd.span_id` in hexadecimal format:
```

```
10.244.0.1 - - [12/Sep/2025:22:41:03 +0000] "GET /health HTTP/1.1" 200 87 0.002 "-" "kube-probe/1.32" "-" dd.trace_id="68c4a17f000000009aed12af2ccb1ead" dd.span_id="9aed12af2ccb1ead"
```

> This means you'll need to remap them as decimal for logs and trace correlation to work in Datadog. 
> 
> To fix this, you'll need to create a pipeline to process the trace ID from your logs. These steps are the same for both instrumentation methods.

Reviewer: ```suggestion
To enable log and trace correlation in Datadog, configure a log processing pipeline to convert these IDs from hexadecimal to decimal. Follow the same steps regardless of the instrumentation method you use.
```
